src/twisted/web/wsgi.py

```diff
@@ -290,6 +290,7 @@ def __init__(self, reactor, threadpool, application, request):
         self.environ = {
             "REQUEST_METHOD": _wsgiString(request.method),
             "REMOTE_ADDR": _wsgiString(request.getClientAddress().host),
+            "REMOTE_PORT": _wsgiString(str(request.getClientAddress().port)),
```

Comment:
Minor comment. have something like `remotePeer = request.getClientAddress()` so that we call it once.

Reply:
Thanks @adiroiban, are you suggesting we add that in addition? i.e. still keep the existing `REMOTE_PORT` and `REMOTE_ADDR` and then also add:
```
"REMOTE_PEER": _wsgiString(request.getClientAddress()),
```

Reply:
My suggestion was something like this

```

peer = request.getClientAddress()
self.environ = {
    "REQUEST_METHOD": _wsgiString(request.method),
    "REMOTE_ADDR": _wsgiString(peer.host),
    "REMOTE_PORT": _wsgiString(str(peer.port)),
}

```